editor/store/effects.js

```diff
@@ -278,19 +284,16 @@ export default {
 
 		// Parse content as blocks
 		let blocks;
+		let isValidTemplate = true;
 		if ( post.content.raw ) {
 			blocks = parse( post.content.raw );
+			isValidTemplate = (
```

Comment:
Wonder if we should mention here we only validate templates that are defined as "locked".